Scripts/RemediationScripts/Remediate-InvalidAADObjectRoleAssignments.ps1

```diff
@@ -543,20 +550,26 @@ class ClassicRoleAssignments
         return($content)
     }
 
-    [PSObject] DeleteClassicRoleAssignmnets([string] $armUri, [string] $method, [psobject] $headers)
+    [PSObject] DeleteClassicRoleAssignmnets([string] $roleAssignmentId, [bool] $isServiceAdminAccount)
```

Comment:
Typo in DeleteClassicRoleAssign**mne**ts.

Reply:
Corrected.